source/assets/public/index.css

```diff
@@ -186,35 +192,59 @@ body {
     vertical-align: middle;
   }
 
+/*Body text defaults*/
+
+p {
+  font-size: 2rem;
```

Comment:
Reorganising these defaults as the way they were organised before was overly specific and making it difficult to style child `p` elements differently. Happy to discuss